sunshine/process.cpp

```diff
@@ -189,6 +190,33 @@ std::vector<ctx_t> &proc_t::get_apps() {
   return _apps;
 }
 
+/// Gets application image from application list.
+/// Returns default image if image configuration is not set.
+/// returns http content-type header compatible image type
+std::tuple<std::string, std::string> proc_t::get_app_image(int app_id) {
+  auto app_index = app_id -1;
+  if(app_index < 0 || app_index >= _apps.size()) {
+    BOOST_LOG(error) << "Couldn't find app with ID ["sv << app_id << ']';
+    return { SUNSHINE_ASSETS_DIR "/box.png", "png" };
+  }
+
+  auto app_image_path = _apps[app_index].image_path;
+  if (app_image_path.empty()) {
+    return { SUNSHINE_ASSETS_DIR "/box.png", "png" };
+  }
+
+  auto image_extention = std::filesystem::path(app_image_path).extension().string();
+  image_extention = image_extention.substr(1, image_extention.length() - 1);
+
+  std::error_code code;
+  if (!std::filesystem::exists(app_image_path, code) || !CHECK_EXPECTED_PICTURE_EXTENTIONS(image_extention)) {
+    return { SUNSHINE_ASSETS_DIR "/box.png", "png" };
+  }
+
+  // return only "content-type" http header compatible image type.
+  return { app_image_path, image_extention == "jpg" ? "jpeg" : image_extention };
```

Comment:
```cpp
return { app_image_path, image_extension };
```

Typos here + limit to PNG format

Reply:
This method will only return a valid file path. Only png files will be returned.

I don't like having code that may be used. The future change, if Moonlight supports jpg, is pretty simple.